ruby/lib/ci/queue/configuration.rb

```diff
@@ -39,14 +40,19 @@ def initialize(
         @flaky_tests = flaky_tests
         @statsd_endpoint = statsd_endpoint
         @grind_count = grind_count
+        @max_duration = max_duration
```

Comment:
You got a bug here. Since you don't go though the setter, `CircuitBreaker::Timeout` won't be appended to the list.

Reply:
opened a pr with the fix. Thanks :)